src/test/java/org/fundacionjala/coding/Raul/test/RegularMovieTest.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.Raul.test;
+
+import org.fundacionjala.coding.raul.movies.RegularMovie;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class RegularMovieTest {
+    private static final int PRICE = 30;
+    public static final double INCREMENT = 1.5;
+
+    private final RegularMovie regularMovie = new RegularMovie("The king", PRICE);
+
+    @Test
+    public void statement() {
+        double thisAmount = 0;
+        thisAmount += INCREMENT;
+        Assert.assertTrue(thisAmount == INCREMENT);
```

Comment:
You can use an assertEquals instead.

Reply:
With double values assertEquals is deprecated. This is the reason because I've used this assert.